### DISCUSSION TITLE
V7 release date

### DISCUSSION THREAD

**Discussion by anton-liam** [sequelize/sequelize#16521] [Help & Questions]
Hi there, is there a release date for v7?

**Comment by WikiRik** [comment]
Not at this moment. We should release a first beta soon, and after some projects have successfully converted to the beta we will release v7. But the v7 alphas currently are already quite stable and can be used in non-production environments.

**Reply by GordonHPepBoys** [reply]
Hi, 
Has been any update? Any release plan?

**Reply by WikiRik** [reply]
No real release plan, but we've discussed this topic with the maintainers recently; https://github.com/sequelize/meetings/wiki/Meeting-2024%E2%80%9003%E2%80%9022#latest-v7-release-plan

**Comment by IsaacInsoll** [comment]
Hi, is it possible to get an update on Sequelize v7 roadmap. Specifically:
- How stable is the alpha?
- When can we expect a beta?
- When do you expect a final release to happen?

I'm currently running a `sequelize` and `sequelize-typescript` project and am tempted by the better DX of something like `prisma` or `drizzle` but I'd rather try migrating to sequelize 7 first to see if solves my current problems. I have no idea if that is a month or 10 years away. 

As a fellow developer: i'm not expecting an exact date, but I have no idea what to expect even roughly.

**Comment by sabinevidal** [comment]
Hi @WikiRik , any update on the roadmap now in 2025? 
I have the same questions as above, specifically related to typescript and being able to use the @sequelize/snowflake package. We'll only be in production by Q2 likely and would be great to know if it's worth already working with the alpha, or if we should stay with v6 for now. 

Thanks!

**Reply by WikiRik** [reply]
No updates unfortunately, we're basically ready to release the first beta but we want to be able to provide more support to the community before we do so.
On the usage of snowflake, in v7 we have made it explicit that support is community driven since we cannot run integration tests. Some things are broken in snowflake because of v7 rewrites, and recently somebody ran the integration tests for us; https://github.com/sequelize/sequelize/pull/17626#issuecomment-2518033251
Help with fixing these errors is really appreciated

**Comment by pitanavvans** [comment]
Hi hope all is well.
Please could you let me know if there is any update on releasing v7.
Thanks

**Reply by IsaacInsoll** [reply]
I switched from sequelize to drizzle about 18 months ago and am loving it. Surely it would be an even easier migration nowadays with LLMs

**Reply by DenisRupp** [reply]
sequelize is dead, unfortunately...

**Reply by WikiRik** [reply]
Not dead, but maintenance is not very active. It's still very suitable to be used
